src/main/java/hackaton/cuckoodobot/Main.java

```diff
@@ -14,6 +21,10 @@
 public class Main {
 
     private final static String BOT_TOKEN = "BOT_TOKEN";
+    private final static String DB_URL = "jdbc:h2:~/cuckoodo;CIPHER=AES"; //db file will be created in home dir with AES encryption
```

Comment:
@rkhozinov I updated URL so db file now will be encrypted with AES and password stored in DB_FILE_PASSWORD env var. Please see http://www.h2database.com/html/features.html#file_encryption for details.